src/test/data/JsonSerializableGuestBookTest/typicalGuestGuestBook.json

```diff
@@ -26,7 +28,8 @@
     "dateRange": "30/12/22 - 02/01/23",
     "numberOfGuests": "3",
     "isRoomClean": "no",
-    "bill": "19.99"
+    "bill": "19.99",
+    "request" : "There are too many insects in my room"
```

Comment:
Should it be phrased as a request instead of a comment?

Reply:
changed